ansible/inventory

```diff
@@ -1,5 +1,91 @@
 [collectors]
-vps770.greenhost.nl
+vps770.greenhost.nl # DELETE discontinued
+
 [probes]
 #211.149.218.16
-87.107.121.133
+87.107.121.133 # DELETE discontinued
+
+[gh:children] # group to gather new-style ACL
+wdc # technically it's i95.net, Radio Free Asia network, but GH has some boxes there
+hkg
+ams
+ath
+ber
+quadix
+tpo
+bigv
+
+[wdc]
+mail.ooni.io # NOT-GH, Art-No-ACCESS - no idea what this is
+mail.test.ooni.io # NOT-GH, Art-No-ACCESS, mail-test.ooni.io - no idea what this is
+ooni-fdroid-build.infra.ooni.io # Art-NO-ACCESS
+a.echo.th.ooni.io # OK (u: root)
+separate.b.collector.test.ooni.io # NOT-GH, Art-NO-ACCESS, OFFLINE?
+separate.bouncer.test.ooni.io # NOT-GH, Art-NO-ACCESS, OFFLINE?
+separate.a.web-connectivity.th.test.ooni.io # NOT-GH, Art-NO-ACCESS, OFFLINE?
+lepidopter-funbox.infra.ooni.io # Art-NO-ACCESS, OFFLINE?
+c.web-connectivity.th.ooni.io # OK (u: root)
+munin.ooni.io # NOT-GH, test-monitoring.ooni.io, Art-NO-ACCESS
+tipitpitap.infra.ooni.io # Art-NO-ACCESS - no idea what this is
+b.collector.ooni.io # OK
+ooni-proxy-dc.infra.ooni.io # OK - MAYBE DISCONTINUE?
+ooni-backend-testing.infra.ooni.io # OK - Temporary machine
+oonibackend-stress-test.infra.ooni.io # OK - MAYBE DISCONTINUE?
+apk-build-test.infra.ooni.ioa # Art-NO-ACCESS - no idea what this is
+
+[hkg]
+datacollector.infra.ooni.io
+# irc-bouncer.service.ooni.io # OK, keep commented for a while, broken UIDs there
+vpn-gate-runner.infra.ooni.io # OK
+seedling.infra.ooni.io # Art-NO-ACCESS - I think this is the machine used to seed lepidopter torrent
```

Comment:
This has been used to seed the lepipodpter torrent, as we decided not offer any torrent we are going to deprecate this host.